serverless/.env.development

```diff
@@ -0,0 +1 @@
+# Add your environment variables here
```

Comment:
Should this file be git ignored?

Reply:
It was meant to be a template for .env with needed values so I fixed this by giving default values.